sys/net/transport_layer/ng_udp/ng_udp.c

```diff
@@ -113,9 +113,10 @@ static void _receive(ng_pktsnip_t *pkt)
     }
     hdr = (ng_udp_hdr_t *)udp->data;
 
+    LL_SEARCH_SCALAR(pkt, ipv6, type, NG_NETTYPE_IPV6);
```

Comment:
How does this perform if we have multiple IP headers? Is it ensured that the upper one is taken to calculate the checksum?